python-sdk/src/astro/sql/operators/base_decorator.py

```diff
@@ -264,11 +266,30 @@ def get_openlineage_facets_on_complete(self, task_instance):
 
         base_sql_query = task_instance.xcom_pull(task_ids=task_instance.task_id, key="base_sql_query")
         job_facets: dict[str, BaseFacet] = {"sql": SqlJobFacet(query=base_sql_query)}
+        collect_source = os.environ.get("OPENLINEAGE_AIRFLOW_DISABLE_SOURCE_CODE", "True")
```

Comment:
Can we add this to the `settings.py` in root like other environment variables? Also we should be doing `conf.get()` from `from airflow.configuration import conf` as there are various ways in airflow to get env.

Reply:
moved to settings